ecs-parent-pom/pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>ecs-parent-pom</artifactId>
-  <version>1.0.1</version>
+  <version>1.0.2-SNAPSHOT</version>
```

Comment:
Something must have gone wrong with the last release build because GitHub wasn't updated to a SNAPSHOT version.